### DISCUSSION TITLE
String problem

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1808] [General]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/6c5c0120-6afb-42eb-b752-b797b0b51a36)

I cant understand why it is not working in my pc!

**Comment by heheboii11** [comment]
ig your string is not formatted correctly, just arrange the full string in a single line. Arrange it and check if it is not working

**Reply by panditdhamdhere** [reply]
Thank you very much problem solved
